tests/sqllogictests/suites/base/09_fuse_engine/09_0021_read_tuple_array

```diff
@@ -41,9 +41,18 @@ SELECT ta[3] FROM t
 NULL
 NULL
 
-## TODO: get from tuple fields
-## SELECT ta[0]:a FROM t
-## SELECT * FROM t where ta[0]:b > 3
+query I
+SELECT ta[1].a FROM t
```

Comment:
Should the syntax be `tuple.field` or `tuple:field`?

Reply:
both are ok.